restapi/src/main/java/io/stargate/web/docsapi/models/BuiltInApiFunction.java

```diff
@@ -0,0 +1,38 @@
+package io.stargate.web.docsapi.models;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.stargate.web.docsapi.exception.ErrorCode;
+import io.stargate.web.docsapi.exception.ErrorCodeRuntimeException;
+import org.codehaus.jackson.annotate.JsonProperty;
+
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
```

Comment:
As long as we only serialize (write) these enums as JSON Objects this is fine; just need to know that deserialization is not supported.

Reply:
This is actually only being serialized at the moment, for the GET built in functions endpoint

When the "execute function" endpoint is requested, the user sends an `operation` and potentially a `value`, and the String value of `operation` is checked against this enum - but it is not actually used in deserialization

Reply:
Ok. And fwtw, we could make it deserialize if we had to, possibly with `@JsonCreator` or if worst came to worst, custom deserializer. So it is not that it can't be done but simply that shape-changing version would be one-way street.